drivers/backup/portworx/portworx.go

```diff
@@ -763,6 +763,7 @@ func (p *portworx) WaitForRunning(
 
 		// Check if backup in progress - stop
 		currentStatus := resp.GetBackup().GetStatus().GetStatus()
+		logrus.Debugf("Current status for backup %s is %s", req.Name, currentStatus)
```

Comment:
do we need this debug line?

Reply:
Well, I am not pretty sure about scheduling. If we have at least one node  that is non-px - it will always be chosen. But in case of testing backup against only single clusuter e.g. in my EKS job it is nice to have torpedo beign running in that cluster, so we can skip this hard  requirement.